documentation/source/model_zoo.md

```diff
@@ -44,22 +44,22 @@ All the available models are listed in the column `Model name`.
-| SSD lite MobileNet v1 | ssd_mobilenet_v1      | COCO    | 320x320    | 24.3                     | **1.55ms**                    | **2.84ms**                          | **8.07ms**                               | **9.14ms**                                        |           **22.76ms**           |
+| Model                 | Model Name            | Dataset | Resolution | mAP<sup>val<br>0.5:0.95  | Latency (HW)*<sub>T4</sub>    | Latency (Production)**<sub>T4</sub> | Latency (HW)*<sub>Jetson Xavier NX</sub> | Latency (Production)**<sub>Jetson Xavier NX</sub> | Latency <sub>Cascade Lake</sub> |                              Torch Compile Supported |
+|-----------------------|-----------------------|---------|------------|--------------------------|-------------------------------|-------------------------------------|------------------------------------------|---------------------------------------------------|:-------------------------------:|-----------------------------------------------------:|
+| YOLO-NAS S            | yolo_nas_s            | COCO    | 640x640    | 47.5(FP16) 47.03(INT8)   | **3.21(FP16)** **2.36(INT8)** |                                     |                                          |                                                   |                                 |                                            Supported |
+| YOLO-NAS M            | yolo_nas_m            | COCO    | 640x640    | 51.55(FP16) 51.0(INT8)   | **5.85(FP16)** **3.78(INT8)** |                                     |                                          |                                                   |                                 |                                            Supported |
+| YOLO-NAS L            | yolo_nas_l            | COCO    | 640x640    | 52.22(FP16) 52.1(INT8)   | **7.87(FP16)** **4.78(INT8)** |                                     |                                          |                                                   |                                 |                                            Supported |
+| PP-YOLOE small        | ppyoloe_s             | COCO    | 640x640    | 42.52                    | **2.39ms**                    | **4.3ms**                           | **14.28ms**                              | **14.99ms**                                       |              **-**              |           TypeError: 'Tensor' object is not callable |
+| PP-YOLOE medium       | ppyoloe_m             | COCO    | 640x640    | 47.11                    | **5.16ms**                    | **7.05ms**                          | **32.71ms**                              | **33.46ms**                                       |              **-**              |           TypeError: 'Tensor' object is not callable |
```

Comment:
Can we just put No/Yes or X/V instead of the error message ?

Reply:
Yep! Fixed that. I left messages mostly for myself (: